integration-tests/config/config.go

```diff
@@ -9,4 +9,22 @@ Enabled = true
 Enabled = true
 ListenIP = '0.0.0.0'
 ListenPort = 6690`
+
+	BaseOCR2VRFTomlConfig = `[Feature]
+LogPoller = true
+
+[OCR2]
+Enabled = true
```

Comment:
is this config only used by all tests? if yes, do we want to use this config for all the other tests ?